packages/dds/tree/src/simple-tree/api/create.ts

```diff
@@ -0,0 +1,179 @@
+	schema: TSchema,
+	data: InsertableTreeFieldFromImplicitField<TSchema>,
+	context?: NodeKeyManager | undefined,
+):
+	| ITreeCursorSynchronous
+	| (TSchema extends FieldSchema<FieldKind.Optional> ? undefined : never) {
+	const flexSchema = toFlexSchema(schema);
+	const schemaValidationPolicy: SchemaAndPolicy = {
+		policy: defaultSchemaPolicy,
+		// TODO: optimize: This isn't the most efficient operation since its not cached, and has to convert all the schema.
+		schema: intoStoredSchema(flexSchema),
+	};
+
+	const mapTree = mapTreeFromNodeData(
+		data as InsertableContent | undefined,
+		schema,
+		context,
+		schemaValidationPolicy,
+	);
+	if (mapTree === undefined) {
+		assert(
+			flexSchema.rootFieldSchema.kind === FieldKinds.optional,
+			"missing non-optional field",
+		);
+		return undefined as TSchema extends FieldSchema<FieldKind.Optional> ? undefined : never;
+	}
+	return cursorForMapTreeNode(mapTree);
+}
+
+/**
+ * Construct tree content compatible with a field defined by the provided `schema`.
+ * @param schema - The schema for what to construct. As this is an {@link ImplicitFieldSchema}, a {@link FieldSchema}, {@link TreeNodeSchema} or {@link AllowedTypes} array can be provided.
+ * @param data - The data used to construct the field content. See `Tree.cloneToJSONVerbose`.
+ * @privateRemarks
+ * This could be exposed as a public `Tree.createFromVerbose` function.
+ */
+export function createFromVerbose<TSchema extends ImplicitFieldSchema, THandle>(
+	schema: TSchema,
+	data: VerboseTreeNode<THandle> | undefined,
+	options: ParseOptions<THandle>,
+): Unhydrated<TreeFieldFromImplicitField<TSchema>>;
+
+/**
+ * Construct tree content compatible with a field defined by the provided `schema`.
+ * @param schema - The schema for what to construct. As this is an {@link ImplicitFieldSchema}, a {@link FieldSchema}, {@link TreeNodeSchema} or {@link AllowedTypes} array can be provided.
+ * @param data - The data used to construct the field content. See `Tree.cloneToJSONVerbose`.
+ */
+export function createFromVerbose<TSchema extends ImplicitFieldSchema>(
+	schema: TSchema,
+	data: VerboseTreeNode | undefined,
+	options?: Partial<ParseOptions<IFluidHandle>>,
+): Unhydrated<TreeFieldFromImplicitField<TSchema>>;
+
+export function createFromVerbose<TSchema extends ImplicitFieldSchema, THandle>(
+	schema: TSchema,
+	data: VerboseTreeNode<THandle> | undefined,
+	options?: Partial<ParseOptions<THandle>>,
+): Unhydrated<TreeFieldFromImplicitField<TSchema>> {
+	const config: ParseOptions<THandle> = {
+		valueConverter: (input: VerboseTree<THandle>) => {
+			return input as TreeLeafValue | VerboseTreeNode<THandle>;
+		},
+		...options,
+	};
+	const schemalessConfig = applySchemaToParserOptions(schema, config);
+	const cursor = cursorFromVerbose(data, schemalessConfig);
+	return createFromCursor(schema, cursor);
+}
+
+/**
+ * Creates an unhydrated simple-tree field from a cursor in nodes mode.
```

Comment:
What does this do if the cursor is not in "nodes" mode? Should we document that?

Reply:
doing so is invalid, will cause undefined behavior which likely depends on the cursor implementation (for our less optimized cursors, you would get a nice error). Most cursor stuff is like that, so I think its fine. This design approach dates back to when our number one priority was minimal overhead on the data access paths so the production cursers have minimal validation and are specced to not require such validation to exist